<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciador do Sistema Greenville</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../sistemaWeb1/back/backendDesenvolvimento/uploads/ico/iconeExcelPrincipal.ico</normaloff>../../sistemaWeb1/back/backendDesenvolvimento/uploads/ico/iconeExcelPrincipal.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(249, 249, 249)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelSystemStatus">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aqui vai o status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ações:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAvancadas">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>8</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>avançadas..</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="btnIniciar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 10px;
background-color: #182e15;
color: white;

border-color: rgb(255, 255, 255);
border-style: solid;
border-width: 1px;

/*
border-color: rgba(255, 255, 255, 0.2);
border-style: solid;
border-width: 0 2px 2px 0;
*/</string>
       </property>
       <property name="text">
        <string>Iniciar</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReiniciar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Reiniciar</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDesligar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Tahoma</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Desligar</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sistema Greenville</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
